specification/src/main/asciidoc/platform/CompatibilityMigration.adoc

```diff
@@ -1,7 +1,7 @@
 == Compatibility and Migration
 
 This chapter summarizes compatibility and
-migration issues for the Jakarta&trade; EE platform. The specifications for each
+migration issues for the Jakarta(TM) EE platform. The specifications for each
```

Comment:
Why does not use Jakarta™?

Reply:
Either approach works...  The current approach of using \&trade; does not...